x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -99,6 +99,15 @@ export type ElasticsearchAssetTypeToParts = Record<
   ElasticsearchAssetParts[]
 >;
 
+export interface DataSet {
```

Comment:
I'd prefer to treat this as one word and use `Dataset` to avoid questions about how to join two words. We're using `Dataset`/`dataset` & `Datasource`/`datasource` in other parts of the code

Reply:
changed.